content/updates/2017-06-12-release-notes.md

```diff
@@ -0,0 +1,25 @@
++++
+date = "2017-06-12"
+title = "Platform Release Notes: June 12, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past week and a half.
+<!--more-->
+
+### Fixed
+
+- We finished migrating all tenants from East/West into GovCloud! Thank you to all our customers for their patience and cooperation in making this move successful.
+- Updated tests, documentation and manual checks we perform to ensure we were ready to begin shutdown of the E/W environment.
+- Clarified documentation about deployer accounts.
+
+### Platform releases
+
+- [Fixed IP addresses for communicating with external apps](https://cloud.gov/docs/apps/static-egress/#cloud-gov-egress-addresses): Our customers often want cloud.gov apps that can talk to the outside and we now have an enabler to make that happen in a more secure way. Outbound traffic from cloud.gov comes from specific IP addresses to help customers open up a connection between cloud.gov and something hosted elsewhere.
```

Comment:
Ooh, there's something interesting here. This should be under its own heading - something like `### Changed` ` rather than `### Platform releases`. We use the "Platform releases" subheading for the general Cloud Foundry updates, rather than for the cloud.gov-specific updates. But this is easy to confuse, since we call cloud.gov a platform - so we probably need a new heading for the CF updates section.

The "Platform releases" section should be pretty similar to the last release notes, including the recommendation to restage apps.